modules/install-sno-ibm-power.adoc

```diff
@@ -0,0 +1,194 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_content-type: PROCEDURE
+[id="installing-sno-on-ibm-power_{context}"]
+= Installing {sno} with {ibmpowerProductName}
+
+.Prerequisites
+
+* You need to setup bastion.
++
+--
+** PXE is used for SNO installation. It requires the following services to be configured and run:
+*** DNS to define api, api-int and *.apps
+*** DHCP service to enable PXE and assign IP to SNO node
+*** HTTP to provide ignition and RHCOS rootfs image
+*** TFTP to enable PXE
+** You will need to install dnsmasq to support DNS, DHCP and PXE, httpd for HTTP.
+--
++
+Perform the following steps to configure bastion to meet these requirements:
++
+. Use the following command to install `grub2`, which is required to enable PXE for PowerVM:
++
+[source,terminal]
+----
+grub2-mknetdir --net-directory=/var/lib/tftpboot
+----
++
+.Example `/var/lib/tftpboot/boot/grub2/grub.cfg` file
+[source,terminal]
+----
+default=0
+fallback=1
+timeout=1
+if [ ${net_default_mac} == fa:b0:45:27:43:20 ]; then
+menuentry "CoreOS (BIOS)" {
+   echo "Loading kernel"
+   linux "/rhcos/kernel" ip=dhcp rd.neednet=1 ignition.platform.id=metal ignition.firstboot coreos.live.rootfs_url=http://192.168.10.5:8000/install/rootfs.img ignition.config.url=http://192.168.10.5:8000/ignition/sno.ign
+   echo "Loading initrd"
+   initrd  "/rhcos/initramfs.img"
+}
+fi
+----
+. Use the following commands to download RHCOS image files from the mirror repo for PXE:
++
+[source,terminal]
+----
+$ export RHCOS_URL=https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.12/latest/
+----
++
+[source,terminal]
+----
+$ cd /var/lib/tftpboot/rhcos
+----
++
+[source,terminal]
+----
+$ wget ${RHCOS_URL}/rhcos-live-kernel-ppc64le -o kernel
+----
++
+[source,terminal]
+----
+$ wget ${RHCOS_URL}/rhcos-live-initramfs.ppc64le.img -o initramfs.img
+----
++
+[source,terminal]
+----
+$ cd /var//var/www/html/install/
+----
++
+[source,terminal]
+----
+$ wget ${RHCOS_URL}/rhcos-live-rootfs.ppc64le.img -o rootfs.img
+----
+
+. To create the ignition file for SNO, you need to create the `install-config.yaml` file:
+
+.. Use the following commands to create the work directory to hold the file:
++
+[source,terminal]
+----
+mkdir -p ~/sno-work
```

Comment:
This is two commands in one, and it doesn't work well for users copying the command. It should be two separate commands. Also it's missing $:

```suggestion
$ mkdir -p ~/sno-work
```